From 557590f09941659a8f78279ab0b6c81b1c81abfe Mon Sep 17 00:00:00 2001
From: someone5678 <someone5678@users.noreply.github.com>
Date: Sat, 30 Sep 2023 18:33:31 +0900
Subject: [PATCH 1/4] telephony: IccPhoneBookInterfaceManager: Change some
 methods to protected/public

* Required for telephony injection in some devices
* This follows QCom's modification

Change-Id: I39e463de389ed15a04757e8b8696b54c8ca06235
Signed-off-by: someone5678 <someone5678@users.noreply.github.com>
---
 .../telephony/IccPhoneBookInterfaceManager.java        | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/java/com/android/internal/telephony/IccPhoneBookInterfaceManager.java b/src/java/com/android/internal/telephony/IccPhoneBookInterfaceManager.java
index ab62aa4ac1..3d7a144d5c 100644
--- a/src/java/com/android/internal/telephony/IccPhoneBookInterfaceManager.java
+++ b/src/java/com/android/internal/telephony/IccPhoneBookInterfaceManager.java
@@ -61,9 +61,9 @@ public class IccPhoneBookInterfaceManager {
     protected static final int EVENT_LOAD_DONE = 2;
     protected static final int EVENT_UPDATE_DONE = 3;
 
-    private static final class Request {
+    public static final class Request {
         AtomicBoolean mStatus = new AtomicBoolean(false);
-        Object mResult = null;
+        public Object mResult = null;
     }
 
     @UnsupportedAppUsage
@@ -388,7 +388,7 @@ public class IccPhoneBookInterfaceManager {
     }
 
     @UnsupportedAppUsage
-    private int updateEfForIccType(int efid) {
+    protected int updateEfForIccType(int efid) {
         // Check if we are trying to read ADN records
         if (efid == IccConstants.EF_ADN) {
             if (mPhone.getCurrentUiccAppType() == AppType.APPTYPE_USIM) {
@@ -398,11 +398,11 @@ public class IccPhoneBookInterfaceManager {
         return efid;
     }
 
-    private String[] getEmailStringArray(String str) {
+    protected String[] getEmailStringArray(String str) {
         return str != null ? str.split(",") : null;
     }
 
-    private String[] getAnrStringArray(String str) {
+    protected String[] getAnrStringArray(String str) {
         return str != null ? str.split(":") : null;
     }
 
-- 
2.34.1

